.github/workflows/anchor.yml

```diff
@@ -25,9 +25,17 @@ jobs:
       - name: Install Anchor
         working-directory: ./staking
         run: npm i -g @coral-xyz/anchor-cli@0.27.0
+      - name: Build anchor (verifiable)
+        working-directory: ./staking
+        run: |
+          npm run build_wasm
```

Comment:
Any reason why these commands are not in the script `test:ci` ?

Reply:
yeah it makes sense to do that. Fixed.